packages/server/lib/browsers/utils.ts

```diff
@@ -419,6 +419,18 @@ const listenForDownload = () => {
   }
 }
 
+const getNetworkEnableOptions = (protocolManager: ProtocolManagerShape | undefined) => {
```

Comment:
could this be a method on the protocol manager itself?

Reply:
Good call: [`0d5cad4` (#28105)](https://github.com/cypress-io/cypress/pull/28105/commits/0d5cad4fe2c7e55e625e54d410ba14aceed04f60)